content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -97,9 +97,10 @@ NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeClaimResize,Defaul
 
 ## What does each admission controller do?
 
-### AlwaysAdmit (DEPRECATED) {#alwaysadmit}
+### AlwaysAdmit {#alwaysadmit} {{< feature-state for_k8s_version="v1.13" state="deprecated" >}}
 
-Use this admission controller by itself to pass-through all requests. AlwaysAdmit is DEPRECATED as no real meaning.
+This admission controller allows all pods into the cluster. It is deprecated because it may not be useful as an admission controller since it's behavior
```

Comment:
Replace L102-103 with:
> This admission controller allows all pods into the cluster. It is deprecated because its behavior is the same as if there were no admission controller at all.